layouts/partials/css/main.css

```diff
@@ -523,6 +523,9 @@ div.main .content .markdown h6 {
   margin-bottom: 1rem;
   letter-spacing: none;
 }
+pre {
```

Comment:
Required for displaying the horizontal scrollbars in code snippets that exceed the maximum width. The change is necessary only for the Chroma case, but it does not break anything in the highlighting.js case.